chadjones839/Capstone
# Registration & Edit Feature

# Description
- New users can register an account with the option of registering as employer or job seeker.
- Registration page logs user in with session storage and routes to appropriate Discovery page.
- Application views for Discovery and Profile use conditionals to check for session storage + account type to open the employer dashboard or candidate dashboard.
- Both candidates and employers can read their profile details.
- Both candidates and employers can edit their profile details.


## Type of change


# Testing Instructions
- Serve sample.json on port 5002
- Click register to go to create new account page.
- Select "looking for work" to create candidate profile, and "looking for talent" for employer profile.
- Fill out fields and click create profile.
- Redirect to discovery page should display company profile cars for candidate users, and vice versa.
- Click the profile icon in the bottom left of navbar.
- User profile should display with the option to edit profile. click the edit profile button.
- Modify the fields then click save changes. Ensure they've been saved in the database.

# Checklist:
